packages/client/src/StreamRegistry.ts

```diff
@@ -181,7 +186,7 @@ export class StreamRegistry implements Context {
         await this.connectToStreamRegistryContract()
         let tx
         if (StreamIDUtils.isENSAddress(domain)) {
-            tx = await this.streamRegistryContract!.createStreamWithENS(domain, path, JSON.stringify(normalizedProperties))
+            tx = await this.streamRegistryContract!.createStreamWithENS(domain, path, JSON.stringify(normalizedProperties), ethersOverrides)
```

Comment:
Is there a way to pass the `ethersOverrides` to the `streamRegistryContract` at once (e.g. constructor or setter) without having to repeat?

Reply:
I wrote it like that so that it could be overridden inside the function call too by adding it to the function signature if needed later...